src/misc/matching.c

```diff
@@ -1018,6 +1018,450 @@ static igraph_error_t igraph_i_maximum_bipartite_matching_weighted(
     return IGRAPH_SUCCESS;
 }
 
+static igraph_error_t igraph_i_maximum_matching_find_aug_path(const igraph_t *graph, igraph_vector_int_t *matching, igraph_vector_int_t *path);
+static igraph_error_t igraph_maximum_matching_unweighted(const igraph_t *graph, igraph_integer_t *matching_size, igraph_vector_int_t *matching);
+static void igraph_i_maximum_matching_get_tree_path(igraph_integer_t *current, igraph_vector_bool_t *f_is_root, igraph_vector_int_t *contraction,
+        igraph_vector_int_t *distance, igraph_vector_int_t *f_parent, igraph_vector_int_list_t *b_lists, igraph_vector_int_t *path);
+
+/**
+ * \function igraph_maximum_matching
+ * \brief Calculates a maximum matching in a graph.
+ *
```

Comment:
```suggestion
 *
 * \experimental
 *
```

Just so we don't forget.